OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -21,6 +22,12 @@ public interface ISpriteLoader
 		bool TryParseSprite(Stream s, out ISpriteFrame[] frames);
 	}
 
+	public interface ISpriteLoader2 : ISpriteLoader
```

Comment:
The way this is implemented is still quite specific to the d2 icn / map format, so I don't think we gain anything by pulling it into the engine code.  For now, it would be better to do test for and open the map file from inside the ICN parser.

If we find another case in the future that needs multiple streams we will be in a better position to implement a more general system that covers all the use cases.